packages/client/src/StreamrClient.ts

```diff
@@ -344,9 +346,14 @@ export class StreamrClient {
      *
      * @param term - a search term that should be part of the stream id of a result
      * @param permissionFilter - permissions that should be in effect for a result
-     */
-    searchStreams(term: string | undefined, permissionFilter: SearchStreamsPermissionFilter | undefined): AsyncIterable<Stream> {
-        return this.streamRegistry.searchStreams(term, permissionFilter)
+     * @param sort - sorting options, default is asc by stream id field
+     */
+    searchStreams(
+        term: string | undefined,
+        permissionFilter: SearchStreamsPermissionFilter | undefined,
+        sort: { sortBy: StreamSortOptions, sortDirection: SortDirection } = DEFAULT_STREAM_SORT
```

Comment:
We could rename this to `order`, `orderBy` and `orderDirection` so that we are consistent with the naming (all arguments are nouns).

(also the field is `orderBy` in The Graph)